src/ja/worker/proxy/proxy.py

```diff
@@ -0,0 +1,40 @@
+from ja.common.proxy.proxy import ContinuousProxy
+from ja.common.message.server import ServerResponse
+from ja.common.proxy.ssh import SSHConfig
+
+
+class WorkerServerProxy(ContinuousProxy):
+    """
+    Proxy for the central server used on the worker client.
+    """
+    def __init__(self, ssh_config: SSHConfig):
+        """!
+        @param ssh_config: Config for paramiko
+        """
+
+    def register_self(self) -> ServerResponse:
+        """! Adds the worker client to the worker client pool of the central
```

Comment:
Sometimes you start the documentation on a new line, sometimes not. Why?

Reply:
No particular reason, I just didn't pay attention.